src/controls/BooleanControl.tsx

```diff
@@ -1,10 +1,18 @@
 /* eslint-disable @typescript-eslint/no-unsafe-assignment */
-import { ControlProps, isDescriptionHidden } from "@jsonforms/core"
+import {
+  ControlElement,
+  ControlProps as JSFControlProps,
+  isDescriptionHidden,
+} from "@jsonforms/core"
 import { Form } from "antd"
 import { Checkbox } from "../antd/Checkbox"
-import { QuestionCircleOutlined } from "@ant-design/icons"
+import { ControlUISchema } from "../ui-schema"
 import { withJsonFormsControlProps } from "@jsonforms/react"
 
+type ControlProps = JSFControlProps & {
+  uischema: ControlUISchema<unknown> | ControlElement
```

Comment:
This reads a little clearer if you reference ControlElement from JSFControlProps:
```
type ControlProps = Omit<JSFControlProps, "uischema"> & {
  uischema: ControlUISchema<unknown> | JSFControlProps["uischema]
```

Reply:
This is how I started and then decided against it 😆 ✅